.gitignore

```
@@ -1,3 +1,5 @@
 *.~vsd*
 *_webDownload/
 .DS_Store
+gistCore.ttl.orig
+docs/ReleaseNotes.md.orig
```

```suggestion
*.orig
```
will prevent more editing later.